js/widgets/page.js

```diff
@@ -130,24 +109,23 @@ $.widget( "mobile.page", {
 		}
 	},
 
-	_setOptions: function( o ) {
-		if ( o.theme !== undefined ) {
-			this.element.removeClass( "ui-page-theme-" + this.options.theme ).addClass( "ui-page-theme-" + o.theme );
-		}
-
-		if ( o.contentTheme !== undefined ) {
-			this.element.find( "[data-" + $.mobile.ns + "='content']" ).removeClass( "ui-body-" + this.options.contentTheme )
-				.addClass( "ui-body-" + o.contentTheme );
+	_setOption: function( optionKey, optionValue ) {
```

Comment:
these names seem a little verbose to me why don't we just use key and value or name and value or something